packages/hdwallet-keepkey/src/keepkey.ts

```diff
@@ -62,7 +62,7 @@ import * as Eth from "./ethereum"
 import { KeepKeyTransport } from "./transport";
 
 export function isKeepKey(wallet: HDWallet): wallet is KeepKeyHDWallet {
-  return typeof wallet === 'object' && (wallet as any)._isKeepKey === true
+  return isObject(wallet) && (wallet as any)._isKeepKey === true
```

Comment:
Would it make sense to type case the rest of the `wallet` to `HDWallet` instead of any as well?

Reply:
that would make sense but i would be worried about introducing side-effects to other wallets, maybe separate PR ?